run_trial.bash

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+#TODO: RE-ENABLE THIS. (dhood): it was disabled because when the server container shuts down, it appears to return non-zero.
+# set -e
+
+TEAM_NAME=$1
+TRIAL_NAME=$2
+
+#TODO: generate unique container names based on input arguments
+CONTAINER_NAME=ariac-server-system
+
+HOST_LOG_DIR=`pwd`/logs/${TEAM_NAME}/${TRIAL_NAME}
+echo "Creating directory: ${HOST_LOG_DIR}"
+mkdir -p ${HOST_LOG_DIR}
+
+# TODO: don't rely on script being run in the root directory
+# TODO: error checking for case when files can't be found
+TEAM_CONFIG_DIR=`pwd`/${TEAM_NAME}
+echo "Using team config: ${TEAM_CONFIG_DIR}/team_config.yaml"
+COMP_CONFIG_DIR=`pwd`/comp_configs
+echo "Using comp config: ${COMP_CONFIG_DIR}/${TRIAL_NAME}.yaml"
+
+LOG_DIR=/ariac/logs
+
+# Create the network for the containers to talk to each other.
+./ariac-competitor/ariac_network.bash
+
+# Start the competitors container and let it run in the background.
+#TODO: parameterize the container name
+./ariac-competitor/run_competitor_container.bash &
```

Comment:
This part of the script is where I am most looking for feedback. It definitely would be preferable to have the competitor code start after the ariac server, so that we don't need to do [waiting like this](https://github.com/osrf/ariac-docker/pull/6/commits/aa6899f2992350e1c4ced7c270d25e4788d0548d#diff-92694b972d13716f9b231735929abe9bR6) in the competitor code.

The reason the competitor code is launched first is for the flow of the script. I launch the competitor code in the background first, and it may run indefinitely. The server, however, will only run for the duration of the trial (limited by a timeout): I launch it in the foreground, it is blocking, and when it ends, we know the trial is finished, so we kill the competitor container.

I am "utilising" the blocking nature of the server here, but it is not ideal, it was just the easiest way I knew how to do it! If you could help me launch the competitor code after the server (and then kill it once the trial has ended), I would like to learn from how you do it.